exporters/common/src/main/java/io/opentelemetry/exporter/internal/InstrumentationUtil.java

```diff
@@ -6,16 +6,15 @@
 package io.opentelemetry.exporter.internal;
 
 import io.opentelemetry.context.Context;
-import io.opentelemetry.context.ContextKey;
-import java.util.Objects;
 
 /**
  * This class is internal and is hence not for public use. Its APIs are unstable and can change at
- * any time.
+ * any time
+ *
+ * @deprecated use {@link io.opentelemetry.context.internal.InstrumentationUtil} instead
  */
+@Deprecated
```

Comment:
[for reviewer] keeping this class allows to keep instrumentation agent compatible as it currently relies on it, switching to the new class location is the next step in the instrumentation agent once this has been merged and released.